config/ibm/vpd_inventory.json

```diff
@@ -22,21 +22,24 @@
                 "keywordName": "DC",
                 "encoding": "DATE"
             }
+        },
+        "xyz.openbmc_project.Inventory.Item": {
+            "PrettyName": {
+                "recordName": "VINI",
+                "keywordName": "DR"
+            },
+            "Present": true
         }
     },
     "frus": {
         "/sys/bus/i2c/drivers/at24/8-0050/eeprom": [
             {
                 "inventoryPath": "/system/chassis/motherboard",
                 "isSystemVpd": true,
-                "systemVpdBackupPath": "/sys/bus/i2c/drivers/at24/7-0050/eeprom",
                 "extraInterfaces": {
                     "xyz.openbmc_project.Inventory.Item.Board.Motherboard": null,
                     "com.ibm.ipzvpd.Location": {
                         "LocationCode": "Ufcs-P0"
-                    },
-                    "xyz.openbmc_project.Inventory.Item": {
```

Comment:
why did you remove PrettyName field here?

Reply:
I didn't remove it's showing difference for other file, I just had vpd_inventory json edited. I'm not sure why will check that.

Reply:
Oh Okay, so the difference here is lesser so, I think git took this. Check with vpd_inventory json will have same things as there.